### PR TITLE
Changes to backend.tf to invoke S3/dynamodb locking

### PR DESCRIPTION
**Pull request by SteveLinden**
This is to add the line git push --set-upstream origin feature/S3-locking-all-envs to all the backend.tf files to force locking on the dynamodb table to stop multiple changes to the terraform.tfstate file if PRs are invoked at the same time on a particular environment.

### CODE REVIEW

**Comment by SteveLinden** [comment]
Comment - removed the environments which would not plan successfully when undertaken manually. Listed in the #2609

**Comment by SteveLinden** [comment]
Added core-network-services, core-shared-services and mi-platform following resolution of the S3 issue listed

**Review by julialawrence — APPROVED** [review]
Looks good to me.

File: terraform/environments/remote-supervision/backend.tf [hunk 1]
```diff
@@ -8,6 +8,6 @@ terraform {
     encrypt              = true
     key                  = "terraform.tfstate"
     region               = "eu-west-2"
-    workspace_key_prefix = "environments/accounts/remote-supervision" # This will store the object as environments/remote-supervision/${workspace}/terraform.tfstate
+    workspace_key_prefix = "environments/accounts/remote-supervision" # This will store the object as environments/remote-supervision/${workspace}/terraform.tfstate    }
```
Comment (ewastempel) [inline terraform/environments/remote-supervision/backend.tf:11]: "}" is this an extra bracket?

**Review by ewastempel — APPROVED** [review]
LGTM

**Pull request merged**